src/messages/Messages.js

```diff
@@ -0,0 +1,50 @@
+import '../App.css';
+import './Messages.css';
+import Filter from '../common/Filter.js';
+import User from '../common/users/User.js';
+import CreateButton from '../common/buttons/CreateButton.js';
+import PostList from '../common/posts/PostList.js';
+import ContactsList from './ContactsList.js';
+
```

Comment:
This is a nitpick about paths -- I recommend almost always using absolute paths instead of relative paths for imports. I only use relative paths when the thing i'm importing lives in the same directory or a subdirectory of my current directory.
Why? It makes things a lot easier when refactoring. If you move a file from place A to place B, and that file uses absolute paths for all its imports, then you don't have to update all the paths of its imports (although you still need to update everything that is importing the file that moved). 
It also makes it easy once your codebase gets large and you need to keep things consistent across files. I can just go to an existing file, copy paste the import path to my new file, and use that same import line in the new file without having to do any path-math.